lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -986,22 +1010,25 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :default => value_of(:disable_memcache_instrumentation),
           :public => true,
           :type => Boolean,
+          :deprecated => true,
           :allowed_from_server => false,
-          :description => 'If <code>true</code>, disables instrumentation for the memcached gem.'
+          :description => deprecated_description(:'instrumentation.memcached', 'If <code>true</code>, disables instrumentation for the memcached gem.')
         },
         :disable_memcache_client => {
           :default => value_of(:disable_memcache_instrumentation),
           :public => true,
           :type => Boolean,
+          :deprecated => true,
           :allowed_from_server => false,
-          :description => 'If <code>true</code>, disables instrumentation for the memcache-client gem.'
+          :description => deprecated_description(:'instrumentation.memcache-client', 'If <code>true</code>, disables instrumentation for the memcache-client gem.')
         },
         :disable_dalli => {
           :default => value_of(:disable_memcache_instrumentation),
           :public => true,
           :type => Boolean,
+          :deprecated => true,
           :allowed_from_server => false,
-          :description => 'If <code>true</code>, disables instrumentation for the dalli gem.'
+          :description => deprecated_description(:'instrumentation.memcache', 'If <code>true</code>, disables instrumentation for the dalli gem.')
         },
         :disable_dalli_cas_client => {
```

Comment:
Should we be updating this config value as well?

Reply:
sorry, I didn't quite understand the question since that description is updated...or is it that I forgot the deprecated => true bit and, if so, then yes, we need that.

Reply:
ooh yeah I meant the `disable_dalli_cas_client` since it didn't have any changes on it